test/integration/client_test.cpp

```diff
@@ -46,10 +47,12 @@ using namespace iroha::ametsuchi;
 using namespace iroha::network;
 using namespace iroha::validation;
 using namespace iroha::model::converters;
+using namespace iroha::model;
 
 class ClientServerTest : public testing::Test {
  public:
   virtual void SetUp() {
+    spdlog::set_level(spdlog::level::off);
```

Comment:
what for?

Reply:
It for tests, in general they shouldn't output anything except ok/not ok